hypothesis-python/tests/array_api/common.py

```diff
@@ -48,3 +52,9 @@ def flushes_to_zero(xp, width: int) -> bool:
         raise ValueError(f"{width=}, but should be either 32 or 64")
     dtype = getattr(xp, f"float{width}")
     return bool(xp.asarray(next_up(0.0, width=width), dtype=dtype) == 0)
+
+
+dtype_name_params = ["bool"] + list(REAL_NAMES)
+dtype_name_params += [
+    pytest.param(n, marks=pytest.mark.xp_min_version("draft")) for n in COMPLEX_NAMES
```

Comment:
I think we need a global `requires_xp_complex = pytest.mark.xp_min_version("draft")` declaration somewhere - it's nicer semantically if we know *why* you need the draft version.  And perhaps more importantly, it'll make it *way* easier to update when that becomes a stable version 😁

Reply:
Went with having a constant as suggested in https://github.com/HypothesisWorks/hypothesis/pull/3456#discussion_r974930884 for now